benchopt/plotting/html/static/result.js

```diff
@@ -136,12 +148,23 @@ const getScatterCurves = () => {
       legendgroup: solver,
       hovertemplate: solver + ' <br> (%{x:.1e},%{y:.1e}) <extra></extra>',
       visible: isVisible(solver) ? true : 'legendonly',
-      x: data(solver).scatter.x,
+      x: data(solver).scatter[ScatterXaxisProperty],
       y: useTransformer(data(solver).scatter.y, 'y', data().transformers),
     });
 
     if (state().with_quantiles) {
       // Add shaded area for each solver, with proper style and visibility.
+
+      solverStoppingStrategy = data(solver)['stopping_strategy'];
+
+      // plot only solvers that were stopped using xaxis type
+      // plot all solver if xaxis type is `time`
+      if(xaxisType !== "time" && solverStoppingStrategy !== xaxisType) {
+        return
+      }
```

Comment:
already done too no? or am I missing a statement?

Reply:
You didn't, thanks for the catch!